.vscode/posix-port-plan.md

```
@@ -0,0 +1,335 @@
+   `LaunchProprietaryBootloader`, `ConfigurationManager_*`, `nanoHAL_Initialize/Uninitialize`,
+   `BlockStorageStream_Read/Seek`, `g_CLR_InteropAssembliesNativeData/Count`.
+
+6. Wired CLR debug output: `posix_stubs.cpp` provides strong overrides for `CLR_Debug::Emit`,
+   `::PrintfV`, and `::Printf` that route to `stdout`.
+
+7. Fixed `extern "C"` linkage for `Events_*`, `FreeManagedEvent`, `CPU_*`, `HAL_AssertEx`,
+   `RtosYield`, `LaunchProprietaryBootloader` in the POSIX implementation files.
+
+### Confirmed output
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)